SampleUserControlLibrary/SampleUserControl.xaml.cs

```diff
@@ -98,11 +98,18 @@ public string SubscriptionKey
             set;
         }
 
+        public string SubscriptionEndPoint
```

Comment:
`endpoint` is one single word, so let us change `EndPoint` to `Endpoint` instead. Also for other places.